integreat_cms/cms/views/utils/hix.py

```diff
@@ -37,7 +39,7 @@ class CacheMeIfYouCan(Exception):
 
 
 @lru_cache(maxsize=512)
-def lookup_hix_score_helper(text: str) -> dict:
+def lookup_hix_score_helper(text: str) -> TextlabResult:
```

Comment:
It appears that this method is only called by the lookup_hix_score method. I assume this is because the method is being cached. If that’s correct, I suggest renaming the method to `lookup_hix_score_cached` or something similar

Reply:
I think the current name is fine, since it is clear from the name that it should not be called. With `lookup_hix_score_cached` this is not the case.